Configuration/TCA/tx_html5mediakit_domain_model_media.php

```diff
@@ -86,6 +86,15 @@
                 'maxitems' => 1,
             ],
         ],
+        'subtitles' => [
+            'displayCond' => 'FIELD:type:=:video',
```

Comment:
Tracks can also be used for `<audio>` Tags, see:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/track

Therefore the `displayCond` can be removed.

Additionally the rendering of the Tracks can be moved to a Partial (e.g. `Tracks.html`) which can then also be added to the [AudioTag](https://github.com/astehlik/typo3-extension-html5mediakit/blob/develop/Resources/Private/Partials/Media/AudioTag.html) Partial.

Reply:
Thank you for bringing this matter to my attention! I actually had some thoughts about it and decided to conduct some research, and I came across this note from **MDN**: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio

`<audio> elements can't have subtitles or captions associated with them in the same way that <video> elements can. See [WebVTT and Audio](https://www.iandevlin.com/blog/2015/12/html5/webvtt-and-audio/) by Ian Devlin for some useful information and workarounds.`

But maybe this is an old note or something! I'll definitely take your suggestions into account and make the necessary changes. Thanks again for taking the time to give me your thoughts!

P.S. I couldn't quote the text from mdn for some reason.

Reply:
@yamenhadla Thank you very much for the feedback. That is a very good point. I only saw, that the `<track>` element can also be used inside `<audio>` elements but i totally missed the hint about browser support.

It seems it is in development but currently only Safari officially supports it:

https://caniuse.com/audiotracks

I would sill recommend the usage of a partial so that it can easily be added to `<audio>` elements later on when the Browser support is available.

I'll leave it up to you if you if you want to add back the display condition for now. Maybe @malteriechmann can share his opinion on this :smiley:

Reply:
I have successfully the tracks functionality within the media class, enabling audio to have tracks as well. I also hinted out in documentation that is not fully supported yet. Like this:

```
==============
Known Problems
==============

There is a minor issue regarding the support of WebVTT track files for audio files. Currently, the majority of web browsers do not fully support this feature.
```

Reply:
I think it is okay to add the tack to `<audio>` as well. It will be supported in the future.